108422 Performance issue when deleting estimate components

rajeshadv opened the pull request:
Files
1. est\u2kinc2c.p
2. est\kstylec.i
3. est\u2kinc1c.p
4. rfq\u2kinc1c.p
5. rfq\u2kinc2c.p
6. sys\inc\round.i
7. sys\inc\xeb-form.i

rajeshadv committed:
108422 Performance issue when deleting estimate components

Files
1. est\u2kinc2c.p
2. est\kstylec.i
3. est\u2kinc1c.p
4. rfq\u2kinc1c.p
5. rfq\u2kinc2c.p
6. sys\inc\round.i
7. sys\inc\xeb-form.i

rajeshadv committed:
108422 - Added logic to move NK1 Round from include to nk1lookup

Files
1. est\kstylec.i
2. est\u2kinc1c.p
3. est\u2kinc2c.p
4. rfq\kstylec.i
5. rfq\u2kinc1c.p
6. rfq\u2kinc2c.p
7. sys\inc\create_nk1.p
8. sys\inc\round.i    (deleted)

rajeshadv committed:
108422 - Performance improvement on eb record delete

Files
1. nosweat\primflds.p
2. est\resetf&b.p
3. sys\inc\xeb-form.i
4. system\session.p

rajeshadv committed:
108422 - Enhancements on create of AuditDtl records

File
1. methods/auditTrigProcs.i

rajeshadv committed:
108422 - Fixed issue with calling the local-open-query when eb record is deleted

Files
1. cec/b-estq.w
2. ce/b-estq.w

rajeshadv committed:
108422 - Fixed issue with displaying fields when the record is deleted

File
1. cec/v-est3.w

ron-advantzware reviewed (changes requested):
change to use table buffer on FOR EACH block

rajeshadv committed:
108422 - xeb-form.i re-factor

Files
1. est\b-multiV.w
2. est\b-multib.w
3. est\d-multbl.w
4. est\EstimateProcs.p
5. est\resetf&b.p
6. sys\inc\xeb-form.i (deleted)
